Azure/sonic-buildimage
# [platform/seastone] Seastone QSFP i2c write function support

#### Why I did it
Enable the SMBus access to QSFP/SFP module. Allow the data to be written to the module's EEPROM.

#### How I did it
Update the Seastone SMBus core driver to support the followings functions:
- SMBus Receive Byte
- SMBus Read/Write Byte Data
- SMBus Read/Write Word Data

#### How to verify it
1. Update the platform driver in SONiC and reboot.
2. Run the bus functionality check on the related adapter.
3. Write the data to the module and read back.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

- Seastone: driver support XCVR EEPROM write.

Test Log:
[dx010-wrtite-access-xcvr-module.log](https://github.com/Azure/sonic-buildimage/files/7025608/dx010-wrtite-access-xcvr-module.log)

#### A picture of a cute animal (not mandatory but encouraged)
